src/HotTyper/test.js

```diff
@@ -39,6 +39,7 @@ class Test extends React.Component {
               'Make your text stylish',
               'Make your text stylish with ZERO effort from you ;).'
             ]}
+            cursor={'I'}
```

Comment:
Shouldn't this be removed as currently the 'I' is the default value or am i missing something?